src/uct/tcp/tcp_ep.c

```diff
@@ -534,6 +536,8 @@ static inline unsigned uct_tcp_ep_sendv(uct_tcp_ep_t *ep, size_t *sent_length)
         if (ctx->comp != NULL) {
             uct_invoke_completion(ctx->comp, status);
         }
+
+        return status;
```

Comment:
return `int`? Also why need return value to be ssize_t?

Reply:
fixed everywhere

Reply:
> return `int`? Also why need return value to be ssize_t?

because `ssize_t send (int __fd, const void *__buf, size_t __n, int __flags);`

`int` may be not enough

Reply:
but it returns result of comparison only, not the length itself

Reply:
Ok, missed that